### PR TITLE
Added Haploid Label Encoder

### PR DESCRIPTION
**Pull request by rahulmohan**
Adds HaploidLabelEncoder class to encoders.py, also added tests and sample data for tests.

### CODE REVIEW

File: variantworks/utils/encoders.py [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import numpy as np
+import torch
+
+def find_insertions(self, base_pileup):
```
Comment (ohadmo) [inline variantworks/utils/encoders.py]: Thanks @rahulmohan, looks much cleaner now. Can you use the refactored method for find_insertions in:
 https://github.com/clara-parabricks/VariantWorks/blob/990f21637419bd492d0ff0d3f1cbf75a1f67f7a9/variantworks/encoders.py#L90-L120
